### ISSUE TITLE
Sidebar not showing anything.

### ISSUE THREAD

**Issue by hadihammurabi** [vuejs/vitepress#330]
### Describe the bug

This is my site with empty sidebar.
![image](https://user-images.githubusercontent.com/16364286/124286102-cbff5980-db78-11eb-90cd-b40c1007bc9e.png)

### Reproduction

- Create a new vitepress project with some directories and md files.
- Run `npx vitepress`

### Expected behavior

Sidebar displays all pages in a tree or how they should be.

### System Info
![image](https://user-images.githubusercontent.com/16364286/124286525-40d29380-db79-11eb-9451-836c8fd8637f.png)
![image](https://user-images.githubusercontent.com/16364286/124286680-65c70680-db79-11eb-8788-d2ad929b8983.png)

### Validations

- [X] Follow our [Code of Conduct](https://vuejs.org/coc)
- [X] Read the [docs](https://vitepress.vuejs.org/).
- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.

**Comment by sosnail** [comment]
@hadihammurabi

I found that the sidebar must have a secondary title.

**Comment by fz6m** [comment]
> @hadihammurabi
> 
> I found that the sidebar must have a secondary title.

Yes, I don't think it's friendly enough
Hope support the display of deeper titles （ like `sidebarDepth` ）

**Comment by newur** [comment]
>     * Create a new vitepress project with some directories and md files.

This sounds like you expect vitepress to show these files / directories automatically - at least that is what I expected. However, vitepress requires a configuration in `.vitepress/config.js` and (mostly) manual setup of the files/dirs. See https://github.com/vuejs/vitepress/blob/c1ba483e587eab1cb7b1dd828c340700d9c01532/docs/.vitepress/config.js#L38 for an example.

I wrote a little plugin to have an automatic way of resolving this, inspired by https://github.com/hannoeru/vite-plugin-pages

Would love to bring my code upstream, but not sure how to best address this, as it is a bit.. rough?

Btw I found a video/stream recording that shows how others struggle with this a well #144 

https://www.youtube.com/watch?v=Gx2aMCwQ3Rg (not affiliated in any way)
